src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Rate/Rate.ts

```diff
@@ -318,6 +321,7 @@ export var directiveFactory = (template : string, adapter : IRateAdapter<RIRateV
                             })
                             .then(() => undefined);
                     }).finally<void>(() => {
+                        scope.isCast = true;
```

Comment:
Ok, so we agree that `m-show-results-click` does not make too much sense in its current form. But we can just leave it in in case it becomes helpful.